core/src/main/java/nl/nn/adapterframework/http/HttpSenderBase.java

```diff
@@ -1202,7 +1202,8 @@ public void setProtocol(String protocol) {
 		this.protocol = protocol;
 	}
 
-	/** If set, the status code of the HTTP response is put in specified in the sessionKey and the (error or okay) response message is returned */
+	/** If set, the status code of the HTTP response is put in the specified sessionKey and the (error or okay) response message is returned.
+	 * A non-okay response is not considered an error which means that the <code>exception</code> forward is not applicable. */
 	public void setResultStatusCodeSessionKey(String resultStatusCodeSessionKey) {
```

Comment:
Does this mean that the resultStatusCodeSessionKey not only stores the return code in the specified session key but it also influences the way errors are handled (nothing is considered an error anymore and forward success is used in every situation)? Can this be explained more explicitly? I had to read it several times to understand what the documentation was trying to say.

Reply:
The described behaviour is exactly what experienced myself and described in the comments of #4487.

Reply:
Lekker dan

Reply:
I have added the sentence "Setting this property has a side effect.". I hope this makes this easier to understand because the reader is warned that he is reading something strange.

Reply:
Je kan beter beschrijven dat wanneer je dit attribuut zet, alle status codes naar de success forward gaan, en responses in de 4xx en 5xx range niet meer als error gezien worden.